lib/Widgets/DatePicker.vala

```diff
@@ -139,6 +136,7 @@ namespace Granite.Widgets {
             Gtk.Allocation size;
             get_allocation (out size);
 
+            rect = new Gdk.Rectangle ();
```

Comment:
The `Gdk.Rectangle` is a struct, you should create it like that:
`rect = Gdk.Rectangle ();`